shared_model/validators/transactions_collection/transactions_collection_validator.cpp

```diff
@@ -28,7 +28,8 @@ namespace shared_model {
             std::shared_ptr<ValidatorsConfig> config,
             TransactionValidator transactions_validator)
         : transaction_validator_(std::move(transactions_validator)),
-          batch_validator_(std::make_shared<BatchValidator>(config)) {}
+          batch_validator_(std::make_shared<BatchValidator>(config)),
+          duplicate_tolerance_(config->duplicate_tolerance) {}
```

Comment:
The meaning is not evident.
Please rename to `txs_duplicates_allowed`

Reply:
Fixed